### DISCUSSION TITLE
View last search?

### DISCUSSION THREAD

**Discussion by cbrake** [LazyVim/LazyVim#1395] [Q&A] [answered]
If I do a live grep (`space-/`), is there any view past greps? I would save some time if I want to repeat a recent grep.

**Comment by dpetka2001** [comment, marked as answer]
There is keybinding `<leader>sR`, which essentially does `:Telescope resume`. Does that cover your needs?

**Reply by cbrake** [reply]
Perfect, thanks!

**Comment by FilipHarald** [comment]
For anyone getting here trying to figure out how to get the history in FZF you can enable like this: https://github.com/LazyVim/LazyVim/discussions/4127#discussioncomment-10535326
